src/modules/timer/Timer.cpp

```diff
@@ -156,5 +156,11 @@ double Timer::getTime()
 #endif
 }
 
+double Timer::getTime()
```

Comment:
I feel like it might make sense to call `getTime` in the Timer module constructor, so it has a consistent start point (for people who load the timer module at least). Thoughts?

Reply:
I thought about doing that, but I wasn't exactly sure where (the Timer module constructor makes a lot of sense). I decided that it probably doesn't matter technically. In regards to the precision problem the difference is small (and probably even a tiny bit better to have it start at first invocation). But maybe one of them is easier to explain and track. I trust your judgment in this.

Reply:
I just notice that the Timer module creates an instance of the Timer class, which calls getTime in its constructor. So do I just have do adjust the comments?

Reply:
Oh, I forgot about that... in that case, just changing the comment to not reference love.run will be sufficient, yeah